src/main/java/uk/gov/dvsa/enums/CertificateTypes.java

```diff
@@ -27,6 +27,8 @@ public enum CertificateTypes {
     CVS_PASS_WELSH("CVS/VTP20W"),
     CVS_PASS_BILINGUAL("CVS/VTP20_BILINGUAL"),
     CVS_FAIL("CVS/VTP30"),
+    CVS_FAIL_WELSH("CVS/VTP30W"),
+    CVS_FAIL_WELSH_BILINGUAL("CVS/VTP30_BILINGUAL"),
```

Comment:
Is there any chance you can refactor this to CVS_FAIL_BILINGUAL so it is consistent with the others please?